requirements-win.txt

```diff
@@ -28,6 +28,7 @@ filelock==3.0.12; python_full_version >= "3.6.0" and python_version >= "3.6"
 finviz==1.3.4
 flair==0.7; python_version >= "3.6"
 flatbuffers==1.12
+fredapi==0.4.3
```

Comment:
Thanks for making this change. Do we not need this in `requirements-full.txt`, `requirements-old.txt` as well?

Reply:
It is already in requirements-full.txt. I made an earlier PR that was merged with that change. Not sure about the requirements-old.txt. Is it used?